test/actions/katello/event_queue_monitor_test.rb

```diff
@@ -9,7 +9,7 @@ class Actions::Katello::EventQueueMonitorTest < ActiveSupport::TestCase
     let(:suspended_class) { ::Actions::Katello::EventQueue::SuspendedAction }
     let(:planned_action) do
       action_class.any_instance.expects(:singleton_lock!)
-      action_class.any_instance.expects(:holds_singleton_lock?).returns(true)
+      action_class.any_instance.expects(:holds_singleton_lock?).returns
```

Comment:
This change should be on the line just above :)

Reply:
Mea culpa.